src/main/scala/microsites/MicrositeKeys.scala

```diff
@@ -289,15 +299,10 @@ trait MicrositeAutoImportSettings extends MicrositeKeys {
       _root_.tut.TutPlugin.tutOne(streams.value, r, in, out, cp, opts, pOpts, re).map(_._1)
     },
     makeTut := {
-      Def.sequential(microsite, tut, micrositeTutExtraMdFiles, makeSite, micrositeConfig)
+      Def.sequential(microsite, tut, micrositeTutExtraMdFiles, makeSite)
```

Comment:
Could we remove the `micrositeConfig` task?

Reply:
Do you mean that it could be removed from the list of available tasks, or if this change of removing it from the sequence of this task is ok?

If it's the former one, I'd say that yes, just checked it and it's not being used anymore.

If it's the second one, also yes, that tasks was adding a `config.yml` to the already Jekyll processed site.

Reply:
Yeah, I meant first :)

If we are not using it anymore we could remove it 👍 Thanks